### PR TITLE
feat(email verification): implement email verification page

### PR DESCRIPTION
**Pull request by tejiri4**
#### What does this PR do?
This PR allows a user to verify his/her account on author haven
#### Description of Task to be completed?
- Create responsive pages
- Create actions
- Create reducer
- Write test
#### How should this be manually tested?
- Git clone the repo
- Cd into it
- Run npm run dev 
#### What are the relevant pivotal tracker stories?
[Email Verification](https://www.pivotaltracker.com/story/show/163825229)

### CODE REVIEW

File: src/action/emailVerification/emailVerificationAction.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+export const UPDATE_STATUS_CODE = 'UPDATE_STATUS_CODE';
+
+export const sendConfirmationMailApiCall = email => {
+  return async dispatch => {
+    try {
+      await makeRequest(`/auth/resend-verification-link`, {
+        method: 'POST',
+        body: { email }
+      });
+      dispatch({ type: 'ON_EMAIL_SUBMIT', payload: { error: '' } });
```
Comment (Sammuel09) [inline src/action/emailVerification/emailVerificationAction.js]: Hello Tejiri, nice work on this task.

However, it seems you are expecting a payload of the error object here and in the catch block. I thought this should be like the success data you are expecting.

Comment (tejiri4) [reply src/action/emailVerification/emailVerificationAction.js]: Thanks I removed the error payload but was not expecting any data as the user will be redirected even if an error occurs.

**Comment by Sammuel09** [comment]
LGTM, yet.

**Comment by Makwe-O** [comment]
LGTM, yet

**Pull request merged**